.github/workflows/docker.yml

```diff
@@ -0,0 +1,59 @@
+name: Build nightly docker image
+
+on:
+  schedule:
+    - cron: "30 0 * * *"
+  push:
```

Comment:
To be removed before merge as then schedule will trigger only on the default github branch